pennylane/ops/op_math/prod.py

```diff
@@ -472,6 +473,26 @@ def terms(self):
                 ops.append(factor)
         return coeffs, ops
 
+    @property
+    def coeffs(self):
+        r"""``coeffs`` in ``coeffs, ops = op.terms()``"""
```

Comment:
Might be worth fleshing out these docstrings a little more, and point the user to both `Operator.terms` and `Operator.pauli_rep`.

Reply:
Good point ✅